packages/react-network/src/context.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
 import {Manager, NoopManager} from './manager';
 
-const {Consumer, Provider} = React.createContext<Manager>(new NoopManager());
+const Context = React.createContext<Manager>(new NoopManager());
+
+const Consumer = Context.Consumer;
+const Provider = Context.Provider;
```

Comment:
I think this has the same issue as destructuring, no? I think we need to not export the consumer separately.

Reply:
With further testing, I don't think we need to do anything anymore. I'm going to close this for now.